src/routes/(app)/+page.svelte

```diff
@@ -52,7 +54,17 @@
 
   // Use if not initializing journey in a "context module"
   onMount(async () => {
-    journeyStore.next();
+    // debugger;
+
+    if (suspendedIdParam) {
+      // Resume journey
+      journeyStore.resume(location.href);
+
+    } else {
+      journeyStore.next(undefined, {
```

Comment:
I understand why we are passing `undefined` here right now, but i must say the api is not super desirable this way. I wonder if we could do something about this. 

`journeyStore` could have a `setup` method or a `options` method where we can configure this value and then pass through the tree information?

An api like this is what i'm thinking

```js
else {
  journeyStore
    .configure({ tree: journeyParam || authIndexValue || undefined })
     .next();
     
 ```
 
 I'm wondering if with `typescript` we could then use a `generic` on `configure` or whatever we call it, so that the `step` can be inferred and passed through to subsequent method calls.
 
 This is all very rough in my head but i think we may be able to sketch out something if you understand where i'm going. Think the generics at different levels exercise we did the other week.

Reply:
Yeah, I get what you're saying. I too have been somewhat on the fence about this. As I've been working through this API design, I've been trying to find the right balance between introducing novel patterns and keeping some consistency with the underlying SDK. That way, those that may use the SDK is some projects, but this Login Framework in others (internal ForgeRockers is one such person), they don't have to flip back-n-forth between different `.next()` implementations.

Now, it's also worth mentioning that this is the Login App portion of code, and this API is not exposed to the Widget user. So, for now, this can be considered internal APIs. But, let's talk it out as I get your concern.

I'm also hesitant to add a `.configure()` method when we also have an `.initialize()` method that has to be run first, that accepts a configuration object. We also already have a `.configure()` method here: https://github.com/cerebrl/forgerock-web-login-framework/blob/6c3949cedcecc562312801b2789cf5c5e71e62aa/src/routes/(app)/%2Blayout.ts#L6.

How about this: What if I leverage the existing SDK convention of having a `.start()` method on this object? That would remove the need for the `undefined` step and keep us consistent with the SDK's methods. Does that sound doable?